src/oneNotePicker.scss

```diff
@@ -20,3 +20,8 @@
   float: right;
   margin-right: 5px;
 }
+
+.picker-selectedItem {
+  // Office UI Fabric specifies a hover which has more priority this, hence the !important
+  background-color: #D0D0D0 !important;
```

Comment:
Yeah! We can't modify partner css files and hover is more scoped than non-hover, so I couldn't think of anything else :(